alibi_detect/cd/mmd.py

```diff
@@ -66,6 +70,9 @@ def __init__(
             Whether to already configure the kernel bandwidth from the reference data.
         n_permutations
             Number of permutations used in the permutation test.
+        batch_size_permutations
+            KeOps computes the n_permutations of the MMD^2 statistics in chunks of batch_size_permutations.
+            Only relevant for 'keops' backend.
```

Comment:
One thing to probably think about now rather than later is naming conventions for parameters which are only relevant for certain backends. I suppose relying on docstring explanation is not the worst thing, but if you have a number of different options for each backend it may be cleaner to organize them in a dict, e.g. `keops_options = {'batch_size_permutations': 1000000}`.

On the other hand, it's possible that some parameter may be used for more than 1 backend but not all, in which case the dictionary convention may not be the simplest, nor pre-fixing parameters with the appropriate backend (e.g. `keops_batch_size_permutations` would work here, but not if the parameter applied to >=2 different backends).